src/jit/gentree.h

```diff
@@ -4221,6 +4221,26 @@ struct GenTreeCopyOrReload : public GenTreeUnOp
 #endif
 };
 
+// Represents GT_ALLOCOBJ node
+
+struct GenTreeAllocObj final : public GenTreeUnOp
+{
+    unsigned int         gtNewHelper;   // Value returned by ICorJitInfo::getNewHelper
+    CORINFO_CLASS_HANDLE gtAllocObjClsHnd;
+
+    GenTreeAllocObj(var_types type, unsigned int helper, CORINFO_CLASS_HANDLE clsHnd, GenTreePtr op
+        DEBUGARG(bool largeNode = false)) :
```

Comment:
Do we really need another DEBUGARG() here?

PeterK is currently revisiting all uses of "largeNodes" in our codebase.

It would seem that we should know if this node needs to be large or not and not have every callsite determine this properly.